Drivers/DevicesInterop/GHIElectronics_TinyCLR_InteropUtil.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <TinyCLR.h>
+#include <Device.h>
+
+extern const TinyCLR_Interop_Assembly Interop_GHIElectronics_TinyCLR_Devices;
+extern const TinyCLR_Api_Manager* apiManager;
+
+const void* TinyCLR_Interop_GetManager(const TinyCLR_Interop_MethodData md, int32_t fieldId);
+TinyCLR_Interop_ClrValue TinyCLR_Interop_GetReturn(const TinyCLR_Interop_MethodData md);
+TinyCLR_Interop_ClrValue TinyCLR_Interop_GetArguments(const TinyCLR_Interop_MethodData md, int32_t argIndex);
+TinyCLR_Interop_ClrValue TinyCLR_Interop_GetFieldInMethodData(const TinyCLR_Interop_MethodData md, int32_t fieldId);
+TinyCLR_Interop_ClrValue TinyCLR_Interop_GetFieldInObject(const TinyCLR_Interop_MethodData md, const TinyCLR_Interop_ClrObject* self, int32_t fieldId);
+
+uint64_t TinyCLR_Interop_CurrentTime();
```

Comment:
Remove the time and interrupt functions for now. We will investigate adding them as needed (and likely find and store the pointer to the api in the managed object).